### PR TITLE
Adding netstorage, appsec and siteshield cachers

### PR DESCRIPTION
**Pull request by stuartio**

### CODE REVIEW

**Comment by ccloes** [comment]
This is awesome @stuartio ... thank you.  We will get this reviewed shortly.

File: akamai_appsec.json.example [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[
+{"status":"covered","hasMatchTarget":true,"hostname":"hola.example.com.mx","policyNames":["MainStreetCorporationMexico"],"configuration":{"id":57415,"name":"MainStreetCorporationMexico","version":37}},
```
Comment (grantleehoffman) [inline akamai_appsec.json.example:2]: Can we pretty print these JSON blobs here

Comment (stuartio) [reply akamai_appsec.json.example:2]: The previous examples were not pretty printed, so I left things as-is. I could add this as a feature, though, no problem

Comment (grantleehoffman) [reply akamai_appsec.json.example:2]: @ccloes would that cause issues? was just throwing it out there as it was the only thing that stood out to me looking through the PR

Comment (ccloes) [reply akamai_appsec.json.example:2]: I am ok either way.  But we talked about that today, the first thing I do is format the docs when viewing them.  There won't be any issue with the way they look as long as they are still JSON format.

**Comment by ccloes** [comment]
Tested this out.  Thank you @stuartio

**Pull request merged**